sdk/go/pulumi/context.go

```diff
@@ -667,12 +667,11 @@ func (ctx *Context) Invoke(tok string, args interface{}, result interface{}, opt
 //
 // args and result must be pointers to struct values fields and appropriately tagged and typed for use with Pulumi.
 func (ctx *Context) invokePackageRaw(
-	tok string, args interface{}, packageRef string, opts ...InvokeOption,
+	tok string, args interface{}, packageRef string, options invokeOptions,
```

Comment:
I pulled the `mergeInvokeOptions` out from here and made this take `invokeOptions` instead of the array of option setters, so that the direct form `InvokePackage` can drop `DependsOn`.